manuscript.Rmd

```diff
@@ -1165,6 +1165,29 @@ This analysis plan may contain data-dependent decisions to increase flexibility
 
 ## Limitations
 
+We have to distinguish two kinds of limitations.
+First, some limitations are by design and are unlikely to be changed soon.
+Our workflow does inherit these from the software it relies on and the fundamental design principles these share with the workflow and `repro`.
+Second, there are other limitations that we hope development effort of the open-source community and, concerning `repro`, us will ease.
+
+As a principle, this workflow is incompatible with steps that cannot be automated.
+This principle is at odds with some ingrained habits of researchers to mix and match manual and automated steps in data analysis.
+To allow for automation, many researchers will have to search for alternative software.  
+The automation friendly software we present here has several technical but critical limitations.
+`Git` can track any files (files bigger than 1GB only with extensions) but can only compare text files (with endings like, `.txt`, `.csv`, `.R`,  or `.Rmd`), not binary files (with endings like `.docx`, `.exe` or `.zip`).
+With `R Markdown`, tables and graphics are dynamically generated from code but cannot be crafted by hand.
+`Make` can orchestrate any programmable software but fails to automate software that will only obey a user clicking at the right buttons.
+`Docker` can ship software that runs on Linux and can be automatically installed, but that requirement precludes many commercial or closed source software.
+
+This move away from software that has served researchers well for decades is understandably difficult and presents us with a conundrum.
+On the one hand, we firmly believe that automated reproducibility makes research more productive and collaboration easier.
+But, on the other hand, we expect researchers to invest considerable time in learning new tools and which they then must "force" on their collaborators.
+Two arguments reconcile this apparent paradox.
+First, this change will not happen all at once.
+Automated reproducibility is an ideal that we believe has many advantages, but it is not an all or nothing decision.
+Researchers can pick up one skill at a time and then help their fellow collaborators do the same.
+Second, the upfront investment is one time (and efforts such as `repro` are underway to reduce it) while the complexity and demands of research projects are steadily increasing.
```

Comment:
I don't understand this argument. The investment is one time but sh*t gets steadily more difficult?

IMO the argument is: The investment is one time, but it pays dividends in increased productivity and reliability

Reply:
Yes to sh*t getting more difficult, statistic in 1900 could be done in the head at 1930 with an abacus, in 1980 with a pocket calculator, 2010 I could get away in excel and 2020 almost requires you to have TPU computation across three different programming languages (okay maybe that is 2030).
No kidding, our author lists are getting longer and longer because research steadily requires more specialists. 
Automation is one way to flatten the curve.
But I see that this is maybe not an argument to be made here. I will adopt the dividend metaphor.